.travis.yml

```diff
@@ -108,5 +99,5 @@ script:
 
   # NNUE testing
   - export CXXFLAGS="-O1 -fno-inline"
-  - make clean && make -j2 ARCH=x86-64-modern debug=no optimize=no learn > /dev/null && ../tests/instrumented_learn.sh --valgrind
-  - make clean && make -j2 ARCH=x86-64-modern sanitize=undefined optimize=no debug=yes learn > /dev/null && ../tests/instrumented_learn.sh --sanitizer-undefined
+  - make clean && make -j2 ARCH=x86-64-modern debug=no optimize=no build > /dev/null && ../tests/instrumented_learn.sh --valgrind
+  - make clean && make -j2 ARCH=x86-64-modern sanitize=undefined optimize=no debug=no build > /dev/null && ../tests/instrumented_learn.sh --sanitizer-undefined
```

Comment:
should be debug=yes.

Reply:
last time I remember debug=yes wasn't working due to some issue with UCI::Option assert. I saw that the code around there changed so maybe will work now.

Reply:
yes, was fixed in the commits you merged.